assets/terraform-modules/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -194,7 +194,7 @@ storage:
             -v /opt/bootkube/assets:/assets:ro \
             -v /etc/kubernetes:/etc/kubernetes:rw \
             --network=host \
-            quay.io/kinvolk/bootkube:v0.14.0-helm-ec64535-amd64 \
+            quay.io/kinvolk/bootkube:v0.14.0-helm2-amd64 \
```

Comment:
I dont understand this tag `helm2` whats the reason behind `2` ?

Reply:
It comes after `helm` (1) tag :smile: I wasn't sure if we should bump the version to `v0.14.1` or `v0.15.0` in our fork.

Reply:
and why not use the shorter version of commit sha to denote based on which commit this image was built ?

Reply:
To use defined way of building the images, as explained in https://github.com/kinvolk/bootkube/pull/13#discussion_r514183062.

Reply:
Hmm, interesting. I am fine with this format if it helps up with easily updating our bootkube with upstream.

Reply:
@invidian 
So if I understand correctly, next bump post `helm2` would be something like below:

`quay.io/kinvolk/bootkube:v0.14.0-helm2.0.1-amd64` ?

Reply:
As those are tags only used by Lokomotive, I'd head towards `quay.io/kinvolk/bootkube:v0.14.0-helm3-amd64`. But I'm happy to use different tagging scheme if you can propose something.